alert-database/src/main/java/com/synopsys/integration/alert/database/job/DistributionJobRepository.java

```diff
@@ -61,8 +61,9 @@
                        + "            OR policyFilters.policyName IN (:policyNames)"
                        + "          )"
                        + "    )"
+                       + "GROUP BY jobEntity.jobId ORDER BY jobEntity.createdAt ASC"
```

Comment:
I believe we can remove `DISTINCT` from the beginning of this query now. I would also prefer that the `GROUP BY` and `ORDER BY` statements are on separate lines. Finally, because of _String Concatenation_, we should prefix each line with at least one space character.